dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/persistence/file/batch/BatchFileOrchestrator.kt

```diff
@@ -275,7 +275,7 @@ internal class BatchFileOrchestrator(
 
     private fun freeSpaceIfNeeded() {
         val files = listSortedBatchFiles()
-        val sizeOnDisk = files.sumOf { it.lengthSafe(internalLogger) }
+        val sizeOnDisk = files.sumOf { f: File -> f.lengthSafe(internalLogger) }
```

Comment:
is explicit type needed?

Reply:
I think it helped it resolving a type at some point